proposal/more-math-functions.md

```diff
@@ -76,17 +78,24 @@ length of the `n`-dimensional vector that has components equal to each of the
 inputs. Since the inputs' units may all be different, the output takes the unit
 of the first input.
 
+## Semantics
+
+### Built-in Module Variables
+
+If a variable is provided by a built-in module, it is not modifiable. Any
+attempts to reassign such a variable results in failure.
```

Comment:
If possible, it would be good to word this as an amendment to [the existing spec language for setting variables](https://github.com/sass/sass/blob/master/spec/variables.md#executing-a-variable-declaration). How you do this specifically is up to you; one possible way is to say in prose "add this text in that place", another way is to rewrite the text with your changes highlighted [as in this spec](https://github.com/sass/sass/blob/master/accepted/import-configuration.md#importing-files).

The goal here is to make it easy for a reader to understand what the canonical language spec will look like once this proposal lands. The more ambiguity there is there, the more avenues there are for individual implementations to interpret it differently.

Reply:
> The goal here is to make it easy for a reader to understand what the canonical language spec will look like once this proposal lands.

Thanks, that put it into perspective. Updated the draft accordingly.